src/docs/santiment-queries/liquidation/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Liquidation Overview
+author: Santiment Team
+date: 2023-04-03
+description: Information about the Liquidation project's tables stored in Santiment Queries.
+---
+
+## Overview
+
+Liquidation is a project that aims to track deposits, borrows, repayments, withdrawals, and liquidation actions occurring on lending protocols.
```

Comment:
The overview should start with a couple of sentences about what is a lending pool. Then the different events should be shortly described: what is a deposit, what is a withdrawal  and so on